docs/usage/shoot_updates.md

```diff
@@ -0,0 +1,56 @@
+# Shoot Updates and Upgrades
+
+This document describes what happens during shoot updates (changes incorporated in a newly deployed Gardener version) and during shoot upgrades (changes for version controllable by end-users).
+
+## Updates
```

Comment:
I think the motivation here is to demonstrate that changes in a Gardener version inevitably have impact on Shooot cluster in the next reconiliation?

maybe that can go in a dedicated section. Other than I would structure it more like this:
 
```
When are updates triggered
 - Maintenance operation (k8s and version updates)
-  Shoot spec changed - also can happen delayed when spec rollout during maintenance time only
- triggered manually by annotation 

What happens during update 
- if the Gardener version itself changed (like you described below) 
- when the Shoot spec changed (depends on what changed - in place vs rolling update. As shown below exactly which field cause which operation would be helpful in my opinion)
```

Just a suggestion though.

Reply:
Yep, sounds reasonable, I have done it, please check :)